src/Writers/MarkdownWriter.jl

```diff
@@ -138,6 +138,36 @@ function render(io::IO, mime::MIME"text/plain", node::Documents.EvalNode, page,
     node.result === nothing ? nothing : render(io, mime, node.result, page, doc)
 end
 
+# Select the "best" representation for Markdown output.
+using Base64: base64decode
+function render(io::IO, mime::MIME"text/plain", d::Documents.MultiOutput, page, doc)
+    foreach(x -> Base.invokelatest(render, io, mime, x, page, doc), d.content)
+end
+function render(io::IO, mime::MIME"text/plain", d::Dict{MIME,Any}, page, doc)
+    filename = String(rand('a':'z', 7))
+    if haskey(d, MIME"image/png"())
+        write("$(filename).png", base64decode(d[MIME"image/png"()]))
```

Comment:
I believe this needs some path computation here. The files end up next to the `make.jl` script currently it seems.

Reply:
Yea, I was just looking at that. In this case it is trivial since `page` is an argument to this function, but we don't propagate that info this far for the other writers.